frontend/src/shared/util/hooks.ts

```diff
@@ -53,9 +53,7 @@ export function useBackgroundRefresh (action: () => void, deps: readonly unknown
                 clearInterval(timerId);
             };
         }
-        // We only want to recreate the interval if the condition changes
-        // eslint-disable-next-line react-hooks/exhaustive-deps
-    }, [condition]);
+    }, [callbackAction, condition]);
```

Comment:
callbackAction was originally left out because it it had the potential to create a lot of setIntervals running at the same time. Maybe I am misremembering but I would like for @KyleLilly to chime in here since he worked on this

Reply:
If this is what I remember, then we definitely don't want this back. It can cause a DDOS effect that wrecks the application.

Reply:
It was [this](https://github.com/awslabs/mlspace/pull/79/files#r1592849120). I'll pull down this branch and take a quick look though.

Reply:
Kyle and I looked at it this morning and while it causes the interval to be recreated way too often that doesn't cause additional network calls or re-renders. Until we have a better solution we're going to let my changes stand as what this fixes is more critical than the potential timing issue it reintroduces.